exe/kubernetes-deploy

```diff
@@ -15,18 +15,23 @@ require 'kubernetes-deploy'
 require 'optparse'
 
 skip_wait = false
+template_dir = nil
 ARGV.options do |opts|
   opts.on("-w", "--skip-wait") { skip_wait = true }
+  opts.on("-tDIR", "--template-dir=DIR") { |v| template_dir = v }
   opts.parse!
 end
 
+if !template_dir || template_dir.empty?
+  raise ArgumentError, "--template-dir flag is required"
+end
```

Comment:
The thing is that we want the default to be `config/deploy/$ENVIRONMENT`, so the script would need `environment` argument just to construct the default template dir. IMO adding extra argument (environment) only to construct the template dir is not the best design.

This is the reason why I moved the default `--template-dir` construction to Shipit side where $ENVIRONMENT is available: https://github.com/Shopify/shipit-engine/pull/640/files#diff-01916dce352e57dea198b45614795a9dR17